build-rpi-raspbian/webif.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# start:     	starts a minimalistic web server
+# 	    	shows the status of OpenHABianPi installation
+# restart:   	checks if webserver is running 
+# inst_done: 	create finish message and link to http://$hostname:8080
+# cleanup:	stops the webserver
+#	     	removes all no longer needed files
+
+hostname=openhabianpi
+port=8888 # Port the webserver is listing to
+
+if [ $1 = "start" ]; then
+        mkdir /tmp/webif
+	ln -s /boot/first-boot.log /tmp/webif/first-boot.txt
+        cd /tmp/webif
```

Comment:
This can often become an issue of confusion. Please work with absolute paths and without `cd`

Reply:
now using absolute paths